solutions/guardrails-policies/05-data-location/template.yaml

```diff
@@ -42,19 +42,16 @@ spec:
     - target: admission.k8s.gatekeeper.sh
       rego: |-
         package datalocation
-
         violation[{"msg": message}] {
           asset := input.review.object
           not location_match(asset.spec.location, input.parameters.locations)
           not allowedResource(asset.kind, input.parameters.allowedServices)
           message := sprintf("Guardrail # 5: Resource %v ('%v') is located in '%v' when it is required to be in '%v'", [asset.kind, asset.metadata.name, asset.spec.location, input.parameters.locations])
         }
-
         location_match(str, pattern) {
           contains(str,pattern[_])
         }
-
         allowedResource(resource, allowed) {
           resource == allowed[_]
         }
----
\ No newline at end of file
+---
```

Comment:
was there a rule to prevent file from ending with `---`

Reply:
not that I've seen, it's either on or off, can't specify if it's at the end of the file
so if the [document-start](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start) rule (`---`) is set to forbidden, it will create a lot of false errors because it's used in many files to separate multiple resources